content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
+            \item The referee selects and indicates a hand-signal. The referee must indicate this signal for \qty{10}{\second}. \todo{reduced this time as 15s is unnecessarily long.}
+            \item During that time or within an additional \qty{10}{\second}, the robot must indicate the hand-signal that it has identified by:
+            \begin{enumerate}
+                \item Using its hand(s) to \emph{mirror} the referee's signal. That is, if the referee given a signals for a decision for the ``red team'' using their right hand, the robot should \emph{mirror} this signal using the robot's \emph{left hand}, as the robot is facing the referee.
+                \item Providing an audio phrase the referee's decision, e.g., ```Goal Red Team'''.  The exact wording is up to the team, but should clearly identify the referee's signal.
+                \item (Optional), At the discretion of the GC/TCM developers, TCP messages may also be sent and interpreted to display the outcome of the robot's decision, however, this is not a requirement for the execution of the challenge. \todo{TW: Allowing this to be optional for now}
+            \end{enumerate}
+            \item The length of time taken for the robot to indicate it's interpretation from the referee blowing the whistle is measured (rounded-up to the nearest second).
+            \item If the robot cannot identify the signal, it should remain motionless and provide no audio output. A robot may continue to pose in the same position as the last signal it identified.
+            \item While not providing a hand-signal or using the whistle, the head referee must keep both hands flat and motionless by their side.
+        \end{enumerate}
 
-        This procedure will be continued for in total four times. Each time, the referee chooses a new decision.
+        This procedure will be repeated \textit{five} times. Each time, the referee chooses a new decision. The referee should choose 2 one-hand static signals, 2 two-hand static signals, and 1 dynamic signal.
 
-    \subsubsection{Available Decisions}
+    \subsubsection{Available Hand-Signals}
 
-        For each decision (not all, because some decisions have to be shown on place, e.g., throw in) will be described and pictured.
+        Each hand-signal for the challenge is described and pictured. Note that for the purpose of clarity, these do not necessarily correspond to human soccer hand-signals.
 
         \begin{itemize}
-            \item \textbf{Kick-Off}
+            \item \textbf{Kick-in \textlangle{}colour\textrangle{} Team.}
+            One-handed signal. One arm, extended horizontally in the direction of the half of the field corresponding to the team that receives the Kick-in Free Kick. That is, right arm extended for the ``Blue team'', and left arm extended for the ``Red team''.
+            \todo{I'm not sure if this is consistent with human soccer, this could be inverted}.
```

Comment:
The problem is that in normal soccer the referee is moving on the field. But we have here the assumption that he does not move. So, we must adapt the rules and signals for now. It could be a good extension for next year. To have a moving referee.

Reply:
I'm not sure I understand this. To my knowledge the direction the ref points is always consistent to the signal. For example the side-ref uses the flag in a consistent direction for a thrown-in (and the field ref confirms the signal?).